sample/WebHook-Generic-CSharp/run.csx

```diff
@@ -4,13 +4,18 @@ using System;
 using System.Net;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 public static async Task<object> Run(HttpRequestMessage req, TraceWriter log)
 {
     string json = await req.Content.ReadAsStringAsync();
     Payload payload = JsonConvert.DeserializeObject<Payload>(json);
 
-    return req.CreateResponse(HttpStatusCode.OK, $"Value: {payload.Value}");
+    JObject body = new JObject()
+    {
+        { "result", $"Value: {payload.Value}" }
+    };
+    return req.CreateResponse(HttpStatusCode.OK, body, "application/json");
```

Comment:
For C# we won't do any defaulting - we require the content type to be specified explicitly. We can revisit that later if needed.